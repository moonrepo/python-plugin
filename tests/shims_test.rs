use proto_pdk_test_utils::*;

#[cfg(not(windows))]
generate_global_shims_test!("python-test", ["pip"]);
