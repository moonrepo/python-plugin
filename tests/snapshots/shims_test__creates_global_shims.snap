---
source: tests/shims_test.rs
expression: "std::fs::read_to_string(sandbox.path().join(\".proto/bin\").join(if cfg!(windows)\n                    {\n                    format!(\"{}.cmd\", \"python-test\")\n                } else { \"python-test\".to_string() })).unwrap()"
---
#!/usr/bin/env bash
set -e
[ -n "$PROTO_DEBUG" ] && set -x



exec proto run python-test --  "$@" 


